platform/security/filter/security-filter-csrf/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -20,6 +19,7 @@
         <property name="whiteListContexts">
             <array value-type="java.lang.String">
                 <value>/services[/]?$=GET</value>
+                <value>/services/.*\?wsdl$=GET</value>
```

Comment:
@stustison Would we need to whitelist both the question mark and dot wsdl paths?

Reply:
I changed that to check the queryString.

Reply:
No, we don't expose anything via ".wsdl" (which is incorrect anyways). There were only some extremely old (read: legacy) systems that did it the wrong way.